lib/trino-metastore/src/main/java/io/trino/metastore/HiveMetastore.java

```diff
@@ -67,6 +67,8 @@ default boolean useSparkTableStatistics()
 
```

Comment:
I would drop the "re-add" from commit message and just talk about adding it like it didn't exist before